README.md

```diff
@@ -0,0 +1,106 @@
+# Bookstore API: Enhancing Your Bookstore Experience
+
+Welcome to the Bookstore API project! Our powerful and secure application, developed using Java Spring Boot, aims to enrich online book purchases for all book enthusiasts. We have combined various features and advanced technologies to provide users with a secure, comprehensive and convenient e-commerce platform. Below, we will introduce you to the key aspects and functional capabilities of our application.
+
+## Features:
+
+### Security:
+
+In our application, we use JWT tokens to ensure secure authentication and authorization. This guarantees that only verified users can access sensitive features.
+
+### Books and Categories:
+
+Administrators have the functionality to add and update categories and books in the database. Users can make use of extensive book search functionality by categories, authors, titles, or prices with pagination and sorting.
+
+### Shopping Cart:
+
+Our users can add books to the shopping cart. The cart is fully functional, allowing users to view, edit, and complete purchases before proceeding to checkout.
+
+### Order Management:
+
+Administrators can manage order statuses, and users can create orders, view their previous orders, and their statuses.
+
+## Used Technologies:
+
+- Spring Boot: Dependency and library management through Maven.
+- Spring Data JPA: Executing CRUD operations on data models.
+- Spring Security: Ensuring application and user data security.
+- Spring MVC: Developing a web application using the Model-View-Controller (MVC) pattern for better code organization and separation of business logic, views, and control.
+- MapStruct: Efficient mapping of models to Data Transfer Objects (DTOs).
+- Docker: Containerization for easy deployment and scalability.
+- OpenAPI: Documenting our RESTful API to provide clear information about available resources, requests, and response formats.
+- Liquibase: Managing and versioning the database schema for easy schema changes and data schema consistency in different environments.
+
+## Functionality:
+
+In our API, functionality is divided for Users and Admin:
+
+| Controller           | User                                                     | Admin                      |
+|-----------------------|----------------------------------------------------------|------------------------------------|
+| BookController        | Book search with filtering, pagination and specification | Read, update, delete functionality |
+| CategoryController    | Category and books by categories search                  | Read, update, delete functionality |
+| AuthenticationController | Registration, authentication                             | Ability to change user roles |
```

Comment:
Please align table

Reply:
It's align in preview mode
![image](https://github.com/keda-vlad/bookstore/assets/135035436/0c5d677d-d1c7-4548-b2a9-a6c1c6edc207)